files/ldap-stg/README.md

```diff
@@ -33,14 +33,6 @@ ldapsearch -LLL -D "cn=admin,cn=config" -W -b "cn=config"
 ldapwhoami -D "uid=testadmin,ou=users,dc=openmrs,dc=org" -W
 >>> testadmin
 
-```
```

Comment:
The new version doesn't generate a self signed cert, so this isn't viable anymore.